simcon_project/users/views/researcher_home.py

```diff
@@ -1,7 +1,22 @@
 from django.shortcuts import render
+from django.db.models import Q
 from conversation_templates.models import TemplateResponse
 from django.contrib.auth.decorators import user_passes_test
-from django.core.paginator import Paginator
+import django_tables2 as tables
+from django_tables2 import RequestConfig
+
+
+class ResponseTable(tables.Table):
+    name = tables.columns.Column(accessor="student.get_full_name")
```

Comment:
We might need to ditch get_full_name and use separate first and last name columns or just email address since when I try to sort the column by alphabetical order by clicking on the column header it seems to not like it. I'll take a look at it a bit and see what I can find first since I think there's no need to separate first and last name if it's possible.